llvm/lib/Transforms/InstCombine/InstCombineCalls.cpp

```diff
@@ -1435,6 +1435,34 @@ static Instruction *foldBitOrderCrossLogicOp(Value *V,
   return nullptr;
 }
 
+Value *InstCombinerImpl::simplifyReductionOperand(Value *Arg,
+                                                  bool CanReorderLanes) {
```

Comment:
I'm happy either way but I suppose at this moment in time this could just be a static function.

Reply:
Done